.github/generate-envs.py

```diff
@@ -126,6 +126,15 @@
         # See also https://github.com/cocotb/cocotb/issues/2300
         "group": "experimental",
     },
+    # Test NVC on Ubuntu
+    {
+        "lang": "vhdl",
+        "sim": "nvc",
+        "sim-version": "master",
+        "os": "ubuntu-latest",
+        "python-version": "3.8",
+        "group": "experimental",
```

Comment:
I just realized: Your PR looks like NVC would pass the cocotb test suite -- is that correct? If so, you can add a non-moving version pointer (either the last release version, of if that's not recent enough, a recent git SHA) to the "ci" group of tests, which will then run for every PR. 

Feel free to keep the master version of NVC in the experimental bucket so we get an early indication if NVC's master breaks.

Reply:
OK, thanks for looking into that. With that many fails it's probably easiest for now to just leave nvc in the experimental group instead of adding explicit excludes. We can do that once we're closer, or once it is clear that certain features are unlikely to be supported by nvc in the short term.